web/src/ApiButton.tsx

```diff
@@ -57,6 +58,14 @@ export const LogLink = styled(Link)`
   font-size: ${FontSize.smallest};
   padding-left: ${SizeUnit(0.5)};
 `
+const ConfirmButton = styled(InstrumentedButton)`
+  && {
+    background-color: ${Color.red};
+    &:hover {
+      background-color: ${Color.red};
+    }
+  }
+`
```

Comment:
there are probably some color contrast a11y issues with this button. i played around a bit and the closest thing i found that i liked was making the text black (for all states) with a dark gray border and light red border on hover/focus/active.

![2021-11-24 13 38 56](https://user-images.githubusercontent.com/23283986/143297046-a6404f3d-6073-4ff2-9913-08def975340d.gif)

here's the code for that:

```scss
&& {
    background-color: ${Color.red};
    border-color: ${Color.gray};
    color: ${Color.black};

    &:hover, &:active, &:focus {
      background-color: ${Color.red};
      border-color: ${Color.redLight};
      color: ${Color.black};
    }
  }

  .fillStd {
    fill: ${Color.black} !important; /* there's some style somewhere with a higher level of specificity than this one that i did not spend time trying to find... */
  }
```

@SurbhiSGupta is welcome to chime in, too! (but we can also address stuff in the design review sesh.)

Reply:
Thanks, that looks much better (and still probably deserves a pass in the design review anyway since since the X wasn't in the figma)